Change pronounced prop to intent in FormFieldLabel (#2007)
jpmorganchase/salt-ds: .changeset/fresh-planets-talk.md

---
"@salt-ds/core": patch
---

> Change `pronounced` prop to `intent` enum. Use "strong" for pronounced label styling

Author: ```suggestion
Change `pronounced` prop to `intent` enum. Use "sentence" for pronounced label styling
```

> Change `pronounced` prop to `intent` enum. Use "sentence" for pronounced label styling

Reviewer: this sounds a bit confusing

Author: updated, lmk

> Change `pronounced` prop to `intent` enum. Use "sentence" for pronounced label styling

Author: ```suggestion
Removed `pronounced` prop to
Added `intent` prop. Use `intent` with "sentence" value to replace pronounced label styling
```

---
"@salt-ds/core": patch
---

> Removed `pronounced` prop to

Reviewer: ```suggestion
Removed `pronounced` prop.
```

> Removed `pronounced` prop.
> Added `intent` prop. Use `intent` with "sentence" value to replace pronounced label styling

Reviewer 2: ```suggestion
The `pronounced` prop has been replaced with the `intent` prop.

```diff
- <FormLabel pronounced />
+ <FormLabel intent="sentence" />
```
